src/Controller/Organization/CommissionableAsset/CommissionableAssetDeleteController.php

```diff
@@ -25,21 +27,21 @@ public function __construct(CommissionableAssetAvailabilityRepository $commissio
         $this->commissionableAvailabilityRepository = $commissionableAssetAvailabilityRepository;
     }
 
-    public function __invoke(EntityManagerInterface $entityManager, CommissionableAsset $commissionableAsset): RedirectResponse
+    public function __invoke(EntityManagerInterface $entityManager, CommissionableAsset $asset): RedirectResponse
     {
         $organization = $this->getUser();
         if (!$organization instanceof Organization || false === $organization->isParent()) {
```

Comment:
this check can be replaced by a security check, maybe in another PR ?

Reply:
Yep. this PR ils Big enough
I was also not sure about the rule: only parents can delete an asset?